src/settings/toml/deploy_target.rs

```diff
@@ -101,10 +121,16 @@ impl DeployTarget {
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum DeployTarget {
-    Zoneless,
+    Zoneless(Zoneless),
     Zoned(Zoned),
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct Zoneless {
+    pub account_id: String,
+    pub script: String,
```

Comment:
nit: i'd call `script` `name` or even `script_name` to remain consistent with the toml